projects/aca-content/src/lib/components/search/search-input/search-input.component.spec.ts

```diff
@@ -55,7 +58,12 @@ describe('SearchInputComponent', () => {
   });
 
   it('should change flag on library400Error event', () => {
+    spyOn(component, 'isSearchRoute').and.returnValue(true);
+    component.ngOnInit();
```

Comment:
don't mix ngOnInit with detectChanges, avoid ngOnInit when testing component behaviour, should be 

```
fixture.detectChanges();
await fixture.whenStable();
```

and unit test marked as `async`, we already have plenty of tests to take a look